edge/pkg/edged/util/record/event_sink.go

```diff
@@ -0,0 +1,85 @@
+package record
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/kubeedge/cloud/pkg/dynamiccontroller/application"
+)
+
+var eventsKind = schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Event"}
+
+type SimpleEventSink struct {
+	*application.Agent
+}
+
+func NewSimpleEventSink(nodeName string) *SimpleEventSink {
```

Comment:
Is there any possible that import this file from kubelet instead of copy in?

Reply:
like kubelet `https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/server.go#L482` I first try `eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface:ed.kubeClient.CoreV1()})` And add a `event.go` to `edge/pkg/metamanager/client/` dir. But I think save event to local storage is unnecessary. And it make the code more complicated.😂